core/sdk-core/src/main/java/software/amazon/awssdk/core/progress/listener/ProgressListener.java

```diff
@@ -189,8 +190,7 @@ default void responseBytesReceived(Context.ResponseBytesReceived context) {
     /**
     * For Expect: 100-continue embedded requests, the service returning anything other than 100 continue
     * indicates a request failure. This method captures the error in the payload
-    * After this, either executionFailure or requestHeaderSent will always be invoked depending on
-    * whether the error type is retryable or not
+    * After this it will either be an executionFailure or a request retry
```

Comment:
Unfinished sentence

Reply:
i meant it as a finishing sentence, modified it to be clearer.